sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -479,9 +464,30 @@ private HttpPipeline createHttpPipeline() {
             .build();
     }
 
-    private TelemetryItemExporter createTelemetryItemExporter() {
-        TelemetryPipeline pipeline = new TelemetryPipeline(builtHttpPipeline);
+    @Nullable
+    private StatsbeatModule initStatsbeatModule(ConfigProperties configProperties) {
+        StatsbeatModule statsbeatModule = null;
+        if (connectionString != null) {
```

Comment:
I missed this in the earlier PR, but need to use `getConnectionString()` here, otherwise env var isn't considered